### ISSUE TITLE
Add Request Management

### ISSUE THREAD

**Issue by Akeru** [hackercouch/hackercouch#24]
It might be required to add some kind of layer between hosts and potential guests regarding communications. For a start maybe add a preferred contact method. On the long term, integrate with some messaging api ?

**Comment by captn3m0** [comment]
I had started out with a free form "contact" field, but later shifted to email because it was easier to implement. We just need to make sure that there is one way of contacting the person (via facebook/email/telegram/whatever).

Setting up a proxy email is not of that much use really, because you have given away your identity by committing itself. The commit already has your email address, so if someone wishes to email you, your email is probably already public.

As for chat, I'm still skeptical if this is needed.

**Comment by ppaeps** [comment]
On 2015-11-30 17:54:49 (+0530), Axel Bodart notifications@github.com 
wrote:

> It might be required to add some kind of layer between hosts and 
> potential guests regarding communications.

The simplicity of "email me to surf my couch" is one of my favourite 
hackercouch features. :)

> For a start maybe add a preferred contact method.

That's not a bad idea.  I've heard rumours that there's an entire 
generation of hackers out there that doesn't use email to communicate.

On the other hand: "please contact me via X to talk about sleeping on my 
couch" probably works fine?

> On the long term, integrate with some messaging api ?

That seems terribly "social" "network"...  Keep it simple?

## 

Philip Paeps
Senior Reality Engineer
Ministry of Information

**Comment by simison** [comment]
> maybe add a preferred contact method.

+1, one or multiple?

> The simplicity of "email me to surf my couch" is one of my favourite
> hackercouch features. :)

+1 keeping it simple. If someone wants to leverage reference or messaging systems, there are a lot of other services there where you can create a profile and simply link from here.

**Comment by captn3m0** [comment]
Yes, my idea is to validate that there is atleast one way to contact the person (via email/twitter whatever). If there is, we shouldn't really care about it much and leave it to the other parties to sort it out.
